packages/terra-date-time-picker/src/terra-dev-site/doc/example/DateTimePickerTimeAttributes.jsx

```diff
@@ -0,0 +1,61 @@
+/* eslint-disable import/no-unresolved */
+import React from 'react';
+import PropTypes from 'prop-types';
+import Field from 'terra-form-field';
+import DateTimePicker from 'terra-date-time-picker/lib/DateTimePicker';
+import DateTimeUtil from 'terra-date-time-picker/lib/DateTimeUtils';
+
+const propTypes = {
+  /**
+   * The current entered date time. Use for the selected date message.
+   */
+  value: PropTypes.node,
+};
+
+const defaultProps = {
+  value: '',
+};
+
+class DateTimePickerExample extends React.Component {
+  constructor(props) {
+    super(props);
+    this.state = { dateTime: this.props.value };
+    this.handleDateTimeChange = this.handleDateTimeChange.bind(this);
+  }
+
+  handleDateTimeChange(event, dateTime) {
+    this.setState({ dateTime });
+  }
+
+  render() {
+    return (
+      <div>
+        <p>
+Selected ISO Date Time:
+          {this.state.dateTime}
+        </p>
+        <Field label="Enter Date/Time" htmlFor="defaultDate">
+          <DateTimePicker
+            name="date-time-picker-example"
+            dateInputAttributes={{ id: 'defaultDate' }}
+            onChange={this.handleDateTimeChange}
+            {...this.props}
+          />
+        </Field>
+      </div>
+    );
+  }
+}
+
+DateTimePickerExample.propTypes = propTypes;
+DateTimePickerExample.defualtProps = defaultProps;
+
+const DateTimePickerTimeAttributes = () => (
+  <DateTimePickerExample
+    value="2017-08-08"
```

Comment:
nit: Unless there is a specific reason to set the default date to `08/08/2017`, it would be more consistent with the other existing examples to use the current date and time as the default value. See https://github.com/cerner/terra-framework/blob/master/packages/terra-date-time-picker/src/terra-dev-site/doc/example/DateTimePickerDefaultDateTime.jsx#L55

Reply:
Updated the example : 7a2559cefa92b922e4683c5f5b547fbef5aac2aa to keep it consistent with other examples.

Reply:
+1 for updating the file name to `DateTimePickerTimeVariant.jsx`

Reply:
done : d239333e941e60599b1818f8447e08fd4fdc10a2